### ISSUE TITLE
Rebalancing of original campaign scenarios

### ISSUE THREAD

**Issue by zenseii** [ihhub/fheroes2#5877]
### Your question

Some of the existing campaign scenarios give ridiculous advantages to the opponents in an effort to compensate for some of the shortcomings of the original game's AI.

The most commonly complained about maps are the 2nd and 7th scenarios of the Price of Loyalty campaign.
These were hard originally too, but will most likely become impossible as the fheroes2 AI improves.

Looking at the 2nd scenario in the editor we can see that red opponent gets 7000 gold and a good amount of resources every week:
<details>

![image](https://user-images.githubusercontent.com/12501091/192089098-c8004c25-835a-4737-8fd4-5f95858ec547.png)

</details>
Yellow opponent gets 100 gold every day plus some resources:

<details>

![image](https://user-images.githubusercontent.com/12501091/192089143-2c652828-0598-4add-bc72-7ad9f64d41f3.png)

</details>

There are also other things like how yellow starts with a hero with 10,000 exp and a castle with buildings up to hydras and level 5 mage guild. For the mage guild there are some maps that become impossible should the opponent randomly get dimension door. This was an issue in the original game too, and the only remedy was to restart the map in this case.

I'm not saying all of these advantages should be removed, but maybe someone has an idea of how to **keep the original intent** of this map while removing some of the "training wheels" that were given to the OG AI.

We already have the difficulty option for campaigns which should help even out the initial resource disparity.

**Comment by mingdong** [comment]
Agreed, like the final stage of the Price of Loyalty campaign(8th scenarios - Corruption's Heart). When over 1.5K bone dragons are divided among enemy heroes as in original game AI,  it is doable task. When enemy forces are all combine with the improved AI with one hero, it becomes mission impossible.

**Comment by TechTrooper** [comment]
In the original campaign Archibald - scenario 7, the enemy AI is attacking me with green dragons as soon as day 4 on normal difficulty.
I've restarted several times and tried different factions, but this seems impossible to win.

**Comment by TechTrooper** [comment]
> > In the original campaign Archibald - scenario 7, the enemy AI is attacking me with green dragons as soon as day 4 on normal difficulty. I've restarted several times and tried different factions, but this seems impossible to win.
> 
> Hi, @TechTrooper. I've seen this map beat several times on latest update by various people. However it's much more doable with the ogre alliance. What bonuses do you have and what difficulty?
> 
> In general until we see that a campaign is definitely impossible to beat will we start considering doing anything, and so far I've seen all maps beat on hard.
> 
> Right now, we are balancing the improvment of the AI with a corresponding removal of AI advantages, one such thing got done yesterday.

I haven't personally compared the mission balance to the original version, but I've watched videos of the original version and I saw players going to capture enemy towns pretty much uncontested.
Couldn't find videos of players facing enemy warlocks with 3 green dragons, 13-18 minotaur kings, 4+ hydras, 50+ centaurs. There's 2 of the warlocks roaming around with similar armies. Can't say for sure if this occurs in the OG or not.

I did the necromancer guild bonus mission and tried starting with def +1 helm and gold bonus, which is not too useful when facing such strong armies right away. Playing on normal difficulty.
